addons/web/views/webclient_templates.xml

```diff
@@ -369,11 +378,16 @@
                 <script type="text/javascript">
                     var odoo = {
                         csrf_token: "<t t-esc="request.csrf_token(None)"/>",
+                        debug: "<t t-esc="debug_mode"/>",
                     };
                 </script>
 
                 <t t-raw="head or ''"/>
             </head>
+            <div class="o_debug_mode text-center text-white" t-if="debug_mode" t-att-debug_mode="debug_mode">
```

Comment:
I know JKE would not agree but I really prefer when the `t-if` is the first attribute... :)

Reply:
Funny cuz I always change the t-if position multiple times since I can't decide which is the best, especially with that kind of class length size..
I'll change it since you are the first one to review, FIFO.